src/Defaults/Trigger/User/UserEmailChanged.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace BracketSpace\Notification\Defaults\Trigger\User;
+
+use BracketSpace\Notification\Defaults\MergeTag;
+
+class UserEmailChanged extends UserTrigger
+{
+	public $changedTo;
+
+	public function __construct()
+	{
+		parent::__construct(
+			'user/email_changed',
+			__(
+				'User email changed',
+				'notification'
+			)
+		);
+
+		$this->addAction(
+			'delete_user_meta',
```

Comment:
Is this the only action you can use? Or maybe WP is firing a specific one when this process happens?

Reply:
I have not seen anywhere that wordpress has any specific action performed at the moment of confirming the email address. Everything happens in the file: wp-admin/user-edit.php.

I was thinking about setting up a hook on "wp_update_user" but here this function can be called programmatically from various plugins and the trigger itself would be executed more often.